docs/operator/installation/olm.md

```diff
@@ -5,88 +5,97 @@ You can install the Starboard operator from [OperatorHub.io](https://operatorhub
 or [ArtifactHUB](https://artifacthub.io/) by creating the OperatorGroup, which defines the operator's
 multitenancy, and Subscription that links everything together to run the operator's pod.
 
-1. Install the Operator Lifecycle Manager:
-
-        curl -sL https://github.com/operator-framework/operator-lifecycle-manager/releases/download/0.16.1/install.sh | bash -s 0.16.1
+As an example, let's install the operator from the OperatorHub catalog in the `starboard-operator` namespace and
+configure it to watch the `default` namespaces:
 
+1. Install the Operator Lifecycle Manager:
+   ```
+   curl -sL https://github.com/operator-framework/operator-lifecycle-manager/releases/download/0.16.1/install.sh | bash -s 0.16.1
+   ```
 2. Create the namespace to install the operator in:
-
-        kubectl create ns starboard-operator
-
+   ```
+   kubectl create ns starboard-operator
+   ```
 3. Declare the target namespaces by creating the OperatorGroup:
-
-
-        cat << EOF | kubectl apply -f -
-        apiVersion: operators.coreos.com/v1alpha2
-        kind: OperatorGroup
-        metadata:
-          name: starboard-operator
-          namespace: starboard-operator
-        spec:
-          targetNamespaces:
-          - default
-        EOF
-
+   ```yaml
+   apiVersion: operators.coreos.com/v1alpha2
+   kind: OperatorGroup
+   metadata:
+     name: starboard-operator
+     namespace: starboard-operator
+   spec:
+     targetNamespaces:
+     - default
+   ```
```

Comment:
I think this is missing the `kubectl apply` command to apply this?

Reply:
Good catch. Added back